src/common/runtime.js

```diff
@@ -187,14 +187,10 @@ export const useNativeTitlebar =
 
 const Env =
   () => {
-    const env = Object.create(null);
     const url = URL.parse(window.location.href);
     const params = url.searchParams;
-    // @Hack: On servo searchParams don't show up (See  servo/servo#10335)
-    if (params == null) {
-      return QueryString.parse(url.search.substr(1));
-    }
-    else {
+    try {
```

Comment:
The only way this would throw is because of the missing `Symbol.iterator`, right? If so, I'd prefer to test for that instead of the `try` block:
```js
if (params[Symbol.iterator]) {
  ..
} else {
  return QueryString.parse(url.search.substr(1));
}
```